bump_version.sh

```diff
@@ -25,9 +27,14 @@ else
       # A temp file is used to provide compatability with macOS development
       # as a result of macOS using the BSD version of sed
       tmp_file=/tmp/version.$$
+      tmp_flake=/tmp/flake.tmp
       sed "s/$old_version_regex/$new_version/" $VERSION_FILE > $tmp_file
       mv $tmp_file $VERSION_FILE
-      git add $VERSION_FILE
+      sed "s/$old_version_regex/$new_version/" $FLAKE_FILE > $tmp_flake
+      mv $tmp_flake $FLAKE_FILE
+      # Run flake update to update the flake.lock file
+      nix flake update
```

Comment:
It was mentioned at some point, but now `nix` is a prerequisite for running `bump_version.sh`.  This should be documented somewhere in the README.

Reply:
Ah, right.  Thanks for remembering that.

Let's go ahead and also add a comment to `bump_version.sh` to that effect.

Reply:
@dav3r yes, it was added to README
@jsf9k Please see https://github.com/cisagov/awssh/pull/27/commits/03443309dc135718053f61e7fa17cece4c4e56ca

Reply:
@jasonodoom I don't see anything in the README about Nix being a prerequisite for running `bump_version.sh` - am I missing something?

Reply:
That works, thanks @jsf9k. 👍